src/plugins/sample-plugin/index.ts

```diff
@@ -7,6 +7,7 @@ import { SampleService } from './services/sample-service';
 import KafkaEventService from '../../event/kafka-event-service';
 import { Queue, Job } from 'bullmq';
 import logger from '../../config/logger.ts';
+import env from '../../config/config.ts';
```

Comment:
Not sure how this would work in a real plugin context. I think we handled this in RC with just each plugin having it's own config. Realistically you should not need to know the connection parameters to just create queues and workers.